exercises/Fibonacci/Python/main.py

```diff
@@ -0,0 +1,14 @@
+n = int(input("Enter an integer :"))

```

Comment:
You should probably add some logic to validate that the input is non-negative (at least for now), might also be nice to ask for a non-negative integer in the prompt.